### DISCUSSION TITLE
How to prevent modal closing when ESC is pressed

### DISCUSSION THREAD

**Discussion by blasferna** [themesberg/flowbite#406] [Q&A] [answered]
I can't find a way to avoid closing the modal when ESC is pressed.

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hey @blasferna,

We'll provide an option for this in the next release, until then here's a quick workaround:

```
const $modaEl = new Modal(modalEl, {
        placement: 'center',
        backdrop: 'static',
        onHide: () => {
            $modaEl.show()
        },
});
```

Not the best solution but it should work if you want to keep it open no matter what.

Cheers,
Zoltan

**Comment by blasferna** [comment]
That sounds great, thanks so much for your help.
